lewis/adapters/stream.py

```diff
@@ -86,7 +84,44 @@ def handle_accept(self):
             StreamHandler(sock, self.target)
 
 
-class Cmd(object):
+class CanNotProcessException(Exception):
+    pass
+
+
+class BaseCommand(object):
+    member = None
+    argument_mappings = None
+    return_mapping = None
```

Comment:
I'm not sure if `return_filter` would tell me much more than `return_mapping`. To me a filter is something that removes something from something else, which is a valid use case for this, but not the only one. Maybe `transformation` was what I had in mind as a use case.

Similar for `argument_mappings` - while most of the time it might be type transformations, it could be other things as well, transforming to lower case, taking a substring, ...

Would `return_transformation` or `argument_transformations` be better? In general I think it would be okay to change it, but only if we find something that is substantially better and captures all possible uses for these two.